snapcraft.yaml

```diff
@@ -1,24 +1,35 @@
 name: classic
-summary: A classic shell environment for Ubuntu Core all-snap installations
-description: This snap provides a full classic shell environment including dpkg and apt support.
-version: 16.04
+summary: A classic shell environment for Ubuntu Core installations
+description: |
+  This snap provides a full classic shell environment including dpkg and
+  apt support.
+version: 18.04-0.1
 confinement: devmode
 architectures: [ all ]
 grade: devel
-assumes: [snapd2.23]
+base: core18
 
 apps:
   create:
     command: bin/create
     plugs: [classic-support]
+    adapter: none
```

Comment:
I assume the adapter is so that we don't get any wrappers but could you please add a rationale for that as a comment in the code.

Reply:
Sure, added. Fwiw, it does not matter if there are wrappers or not it just feels cleaner this way.